src/modules/admin/components/forms/FormDefinitionDetailPage.tsx

```diff
@@ -38,18 +40,18 @@ const FormDefinitionDetailPage = () => {
       MutationUpdateFormDefinitionArgs
     >({
       formRole: StaticFormRole.FormDefinition,
-      initialValues: formDefinition || {},
-      localConstants: { definitionId: formId },
+      initialValues: formIdentifier?.draft || {},
```

Comment:
```suggestion
      initialValues: formIdentifier?.displayVersion || {},
```

I know it will change later with design, but it seems wrong that it's using `draft` for the `initialValues`, but `displayVersion` to determine which form the "submit" button will actually update.  Am I getting that right?

Reply:
Yes this was a mistake - thanks for catching it!